src/transformers/modeling_mobilebert.py

```diff
@@ -44,7 +44,9 @@
 
 
 logger = logging.getLogger(__name__)
-MOBILEBERT_PRETRAINED_MODEL_ARCHIVE_LIST = []
+MOBILEBERT_PRETRAINED_MODEL_ARCHIVE_LIST = [
+    "mobilebert-uncased"
```

Comment:
I don't think this will work – at least, I intended for this to *not work* :)

Should be `"google/mobilebert-uncased"` here @LysandreJik

Reply:
I suggest we ensure there is a reference to "uncased_L-24_H-128_B-512_A-4_F-4_OPT" somewhere even in a comment as shown on the latest Google GitHub Page [here](https://github.com/google-research/google-research/tree/master/mobilebert) It seems Google may have multiple configurations as shown in this [config file](https://github.com/google-research/google-research/tree/master/mobilebert/config). This is just to avoid future confusion.